src/preamble.js

```diff
@@ -910,6 +910,7 @@ function getWasmImports() {
 
 #if '$wasmTable' in addedLibraryItems && !RELOCATABLE
     wasmTable = wasmExports['__indirect_function_table'];
+    usedTableLength = wasmTable.length;
```

Comment:
Ideally we could avoid adding any code here, since it will effect all programs.

You can use the library system to inject some startup code if (and only if) usedTableLength is used.

```
$usedTableLength__postset: () => addAtInit(`usedTableLength = wasmTable.length`);     
```

This way only programs that reference `usedTableLength` will pay the code.

Reply:
I tried this, but I have reverted, as I suspect failing test to be connect with it, but I am not sure.

Reply:
It failed again, I have to setup a more complex dev environment to be able to run browser tests.

Reply:
What test failed?